recipes/busybox/busybox.inc

```diff
@@ -73,6 +73,10 @@ do_compile_binfmt_flat_fixup() {
 
 require busybox-install.inc
 
+PACKAGES:<USE_busybox_udhcpd = "udhcpd-config "
```

Comment:
looking at the rest of the file, this should be "${PN}-udhcpcd-config ", to be consistent